src/Umbraco.Web.UI.Client/src/views/propertyeditors/slider/slider.controller.js

```diff
@@ -75,18 +153,30 @@
         }
 
         //initiate slider, add event handler and get the instance reference (stored in data)
-        var slider = $element.find('.slider-item').slider({
+        var slider = $element.find('.slider-item').bootstrapSlider({

```

Comment:
See https://github.com/seiyria/bootstrap-slider#what-if-there-is-already-a-slider-plugin-bound-to-the-jquery-namespace .. not sure why .slider() didn't seem to work?